sdk/keyvault/keyvault-keys/src/localCryptographyClient.ts

```diff
@@ -39,20 +40,28 @@ export class LocalCryptographyClient {
    * @param plaintext - The text to encrypt.
    */
   public async encrypt(
-    algorithm: LocalSupportedAlgorithmName,
-    plaintext: Uint8Array
+    encryptParameters: EncryptParameters,
+    _options: EncryptOptions = {}
   ): Promise<EncryptResult> {
     if (!isNode) {
       throw new LocalCryptographyUnsupportedError("Encryption is only available in NodeJS");
     }
     const result = (await runOperation(
       this.key,
       "encrypt",
-      algorithm,
-      Buffer.from(plaintext)
+      encryptParameters.algorithm as LocalSupportedAlgorithmName,
+      Buffer.from(encryptParameters.plaintext)
     )) as Buffer;
     const keyID = this.key.kid;
-    return { result, algorithm: algorithm as EncryptionAlgorithm, keyID };
+    return {
+      result,
+      algorithm: encryptParameters.algorithm as EncryptionAlgorithm,
+      keyID,
+      // TODO: is this right? Doesn't feel right since we don't support the AES crypto algorithms locally anyway
+      additionalAuthenticatedData: (encryptParameters as any).additionalAuthenticatedData,
```

Comment:
This really doesn't feel right since we don't support the AES crypto algorithms locally anyway... This codepath should never really be hit, but never can also mean in 2 months... 

Added the code just to have something to attach to my comment though. If nobody says otherwise I'll remove it.

Reply:
Instead of this, can we add an internal mocked test?